gen/dcos-config.yaml

```diff
@@ -1350,6 +1350,22 @@ package:
           "registrar",
           "allocator",
        ]
+      # Reads 'mntr' stats from one or many zookeeper servers
```

Comment:
"Reads Zookeeper 'mntr' health checks from..." would help for those who don't know what `mntr` is without looking.